test/azure/Expected/AcceptanceTests/Lro/lro/aio/operations/_lros_operations.py

```diff
@@ -2174,14 +2174,10 @@ async def begin_delete_provisioning202_accepted200_succeeded(
         kwargs.pop('content_type', None)
 
         def get_long_running_output(pipeline_response):
-            response_headers = {}
```

Comment:
I checked out these missing response headers, and I think we were incorrectly generating them in the first place. Basically what happened was that we were setting the [202 response](https://github.com/Azure/autorest.testserver/blob/master/swagger/lro.json#L762) as the final deserialization, when it should've been the 200 response

Reply:
I've checked all of the response headers where we no longer generate, and this is the case, @lmazuel what're your thoughts? I don't think this is a breaking change since it's really a bug fix, and all of the response headers we exposes before were empty